yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -514,6 +526,11 @@ func renameDbzmExportedDataFiles() error {
 			newFilePath = filepath.Join(exportDir, "data", status.Tables[i].SchemaName+"."+tableName+"_data.sql")
 		}
 
+		if !utils.FileOrFolderExists(oldFilePath) && utils.FileOrFolderExists(newFilePath) { //In case of restarts rename should not be done again else it will error out
```

Comment:
this case is not required anymore because it's handled in `checkAndHandleSnapshotComplete`, correct?